### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by toughpc** [husqin/DnCNN-keras#2]
Why lost function in Keras is not about residual learning R(x)-(x-y) like what is said in the paper

**Comment by shail-1812** [comment]
A residual network aims at taking the previous layers and performs operations such as addition, subtraction etc with the current layer. In this case of DNCNN model the residual network works as follows. 
The input image is stored in a variable input. The model than performs convolution in order to attempt to remove noise from the image. After the layers are added the model takes the last output layer and
subtracts the variable defined earlier as input. This is how the residual network is applied in the following algorithm. Our main aim of the model is to separate the noise from the image and that is the reason
we are subtracting the input layer from the output layer. Subtracting separates the noise from the image as defined in the paper.
